model/schema/affectedGenomicModel.yaml

```diff
@@ -182,7 +185,12 @@ slots:
   parental_populations:
     singular_name: parental_population
     domain: AffectedGenomicModel
-    range: uriorcurie
+    range: AffectedGenomicModel
+
+  parental_population_curies:
+    domain: AffectedGenomicModelDTO
```

Comment:
Reference to an AGM from an AGM is going to enforce loading order constraints (or a two-stage load process).  I guess this is okay, an Association and separate load seems overkill.  Possibly worth discussing with the rest of the A-Team.